src/loan-returned/handler.js

```diff
@@ -13,6 +13,7 @@ module.exports.handle = (event, context, callback) => {
         callback(null, generateSuccessMessage(loanData.item_loan.loan_id))
       })
       .catch(e => {
+        console.log(e)
```

Comment:
Can you look at adding a proper nodejs logging library making use of info, debug etc. The full stack trace `e` should only be done on debug - Generally just a message saying `Failed to retrieve loan X` is better